test/unit/test_reanalysis_downloading_toolkit.py

```diff
@@ -0,0 +1,284 @@
+import pandas as pd
+
+from operational_analysis.toolkits import reanalysis_downloading as rd
+
+
+def test_default_var_dicts_planetos():
```

Comment:
If you go with my comment in the function definition, I'd add a test for a typo like " MERRA2 ". Otherwise, I think if you add a case where it returns `None`, this test is good to go.

Reply:
Expanded the test cases and added a test for the new `_get_dataset_names` function too.

Reply:
Looks good!